### PR TITLE
feat: Add a new bot invite menu.

### PR DESCRIPTION
**Pull request by NoLogicAlan**
This code allows you to add a bot to your Revolt server or group. Simply select the option to add to a server or group, then choose the specific server or group from the dropdown menu. Click the "Add Bot" button to complete the process. This code is easy to use and can help streamline your communication and organization efforts within Revolt.

Before
<p align="left">
  
</p>
After
<p align="left">
  
</p><p align="left">
  
</p><p align="left">
  
</p><p align="left">
  
</p>

## Please make sure to check the following tasks before opening and submitting a PR

* [x] I understand and have followed the [contribution guide](https://github.com/revoltchat/revolt/discussions/282)
* [x] I have tested my changes locally and they are working as intended
* [x] These changes do not have any notable side effects on other Revolt projects
* [x] (optional) I have opened a pull request on [the translation repository](https://github.com/revoltchat/translations)
* [x] I have included screenshots to demonstrate my changes

### CODE REVIEW

**Comment by Rexogamer** [comment]
Closing due to the ongoing rewrite - in addition, we're currently undecided as to what we want to replace the current screen with.

File: src/pages/invite/InviteBot.tsx [hunk 1]
```diff
@@ -38,71 +73,113 @@ export default function InviteBot() {
...
-                        <Button
-                            palette="secondary"
-                            onClick={() =>
-                                group !== "none" &&
-                                client.bots.invite(data._id, { group })
-                            }>
-                            Add
-                        </Button>
+                        <Category>
+                            Add this bot to your Revolt server, group
```
Comment (insertish) [inline src/pages/invite/InviteBot.tsx:114]: Reword -> `Add this bot to your server or group`

**Review by insertish — COMMENTED** [review]
I probably wouldn't mind merging this since while the client is being replaced this is still a bit more user friendly than what we have now

File: src/pages/invite/InviteBot.tsx (hunk 1 above)
Comment (NoLogicAlan) [reply src/pages/invite/InviteBot.tsx:114]: oh yeah that look better

**Comment by NoLogicAlan** [comment]
> I probably wouldn't mind merging this since while the client is being replaced this is still a bit more user friendly than what we have now

Oh yeah, your right i try my best to make it more easer for users like using one button to inv the bot to server or group and changed the options, select for both.

**Pull request closed without merging**